site/content/docs/5.2/layout/columns.md

```diff
@@ -64,6 +76,8 @@ Use flexbox alignment utilities to vertically and horizontally align columns.
 </div>
 {{< /example >}}
 
+Or, change the alignment of each column individually with any of the responsive `.align-self-*` classes.
+
```

Comment:
**[Bootstrap Side]** Shouldn't we introduce those changes to https://deploy-preview-1757--boosted.netlify.app/docs/5.2/layout/columns/#horizontal-alignment as well ?

Reply:
See https://github.com/twbs/bootstrap/pull/37899.